index.html

```diff
@@ -216,10 +216,10 @@ <h3>Book</h3>
           <h3>Community</h3>
           <p>There are several places in which people gather to discuss PureScript:</p>
           <ul style="padding-left: 15px">
-            <li><strong>Slack</strong> -- The #purescript channel on <a href="https://functionalprogramming.slack.com/">FP Slack</a> is an active chat room. Use the <a href="https://fpchat-invite.herokuapp.com/">FP Slack invite bot</a> to make an account there.</li>
-            <li><strong>IRC</strong> -- The <a href="irc://irc.freenode.net/purescript">#purescript channel on IRC Freenode</a>. It is slightly less active than the Slack channel above.</li>
-            <li><strong>Discourse</strong> -- The <a href="https://discourse.purescript.org">PureScript Discourse</a> has become a popular web forum.</li>
-            <li><strong>Regular online gatherings</strong> -- Individual events can be found and  RSVPed at <a href="https://gathering.purescript.org/">gathering.purescript.org</a>, while the schedule is documented in the <a href="https://github.com/purescript/documentation/blob/master/ecosystem/PureScript-Meetups.md">github.com/purescript/documentation project</a>.</li>
+            <li><strong>Slack</strong> -- The #purescript channel on <a href="https://functionalprogramming.slack.com/">FP Slack</a>. Use the <a href="https://fpchat-invite.herokuapp.com/">FP Slack invite bot</a> to make an account there.</li>
+            <li><strong>IRC</strong> -- The <a href="irc://irc.freenode.net/purescript">#purescript channel on IRC Freenode</a>.</li>
+            <li><strong>Discourse</strong> -- The <a href="https://discourse.purescript.org">PureScript Discourse</a> instance.</li>
+            <li><strong>Regular online gatherings</strong> -- Individual events can be found and RSVPed at <a href="https://gathering.purescript.org/">gathering.purescript.org</a>, while the schedule is documented in the <a href="https://github.com/purescript/documentation/blob/master/ecosystem/PureScript-Meetups.md">github.com/purescript/documentation project</a>.</li>
```

Comment:
Sorry, can you please remove "and RSVPed" here

Reply:
Also can you please cut this down a bit further by just saying something like "and the schedule is documented <link>here</link>"